### ISSUE TITLE
RUSTSEC-2021-0073 — prost-types 0.7.0

### ISSUE THREAD

**Issue by jnicholls** [cosmos/cosmos-rust#107]
`cosmos-rust` as well as `tendermint-rs` rely on `prost` and `prost-types` 0.7. The `prost-types` crate has a reported security advisory [RUSTSEC-2021-0073](https://rustsec.org/advisories/RUSTSEC-2021-0073). An upgrade to `prost` and `prost-types` 0.8 would address the advisory. Thanks.

**Comment by tony-iqlusion** [comment]
Yeah, we're aware, and as you've noted are blocked on an upgrade to `tendermint-rs`.

**Comment by jnicholls** [comment]
Yep, thanks @tony-iqlusion. Having this issue gives me something to point for tracking purposes in my work with Cosmos. Appreciate the work you do!
